sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/api/Azure.ResourceManager.DesktopVirtualization.netstandard2.0.cs

```diff
@@ -63,7 +65,6 @@ public partial class HostPoolData : Azure.ResourceManager.Models.TrackedResource
         public Azure.ResourceManager.DesktopVirtualization.Models.HostPoolLoadBalancerType LoadBalancerType { get { throw null; } set { } }
         public Azure.Core.ResourceIdentifier ManagedBy { get { throw null; } set { } }
         public int? MaxSessionLimit { get { throw null; } set { } }
-        public Azure.ResourceManager.DesktopVirtualization.Models.DesktopVirtualizationMigrationProperties MigrationRequest { get { throw null; } set { } }
```

Comment:
This is a breaking change that now allow, please use [custom code](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitiveservices/Azure.ResourceManager.CognitiveServices/src/Customized/Models/ServiceAccountEncryptionProperties.cs) to add this property back and mark it as `Obsolete` and `EditorBrowsableState.Never`

Reply:
Do I need to make any changes to swagger? Or can I just change the code here in the .netstandard2.0.cs file?

Reply:
Swagger change is the better way

Reply:
Ok, let's use the custom code to fix all breaking chagnes